JokeRandomizer/JokeRandomizer/CustomButtons/CustomLabel.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  CustomLabel.swift
+//  JokeRandomizer
+//
+//  Created by john.adrian.de.claro on 12/7/22.
+//
+
+import UIKit
+
+fileprivate let defaultFontSize: CGFloat = 15.0
+
+final class AuthLabel: UIView {
```

Comment:
Where is this class being used? Please don't include unused code from being committed unless this is going to be used in the future. If that will be the case, please leave a descriptive note on top of the file.

Reply:
removed unused codes and file private for future font size. CustomLabel for setup and punchline